lib/dns.c

```diff
@@ -27,21 +27,19 @@ limitations under the License.
 struct dns_entry {
     char hostname[MAX_DNS_NAME];
     char ip[MAX_IP_LENGTH];
-
-    LIST_ENTRY(dns_entry) _next;
 };
 
 typedef struct cache_s {
     uint32_t base;
     uint32_t counter;
     uint32_t counter_bits;
-    LIST_HEAD(entries, dns_entry) entries;
+    model_map entries;
 } cache;
 
 static cache ip_cache = {
-        .base = 0xA9FE0000, // 169.254.0.0
+        .base = 0x64400000, // 100.64.0.0
```

Comment:
saw this in wireguard code recently and figured i'd steal it some day... 
```
(100 << 24) | (64 << 16) | (0 << 8) | (0 << 0)
``` just a thought :)

Reply:
bahh, just learn reading hex